src/engine/controls/loopingcontrol.cpp

```diff
@@ -768,7 +769,7 @@ void LoopingControl::slotLoopEndPos(double pos) {
 }
 
 // This is called from the engine thread
-void LoopingControl::notifySeek(double dNewPlaypos, bool adjustingPhase) {
+void LoopingControl::notifySeek(double dNewPlaypos, double dSyncedNewPlayPos, bool adjustingPhase) {
```

Comment:
Assume an active loop starting exactly at hotcue A. If I activate the hotcue the synced position is very likely in front of the loop. The adjustingPhase is still false, as it is was a SEEK_STANDARD_PHASE. If we set the adjustingPhase to true in that case, the loop would not be disabled for any other seek position outside the loop.
Thus, the loopcontroller should know about the position requested by the user but the remaining controllers should still be notified with the synced position.

I assume that the adjustPhase is still necessary. When there was no sync, both positions are equal. But (I guess) the same holds true if only a sync without any seeking is requested.

Reply:
I just had a thought on solving this case differently. Let the loopingcontroller calculate a position inside a loop (if enabled) based on the requested and the synced position.

```suggestion
    double syncPosition = position;
    if (!paused && (seekType & SEEK_PHASE)) {
        syncPosition = m_pBpmControl->getNearestPositionInPhase(position, true, true);
        position = m_pLoopingControl->getSyncPositionInsideLoop(position, syncPosition);
    }
```
The setNewPos and notifySeek could be left unchanged. Furthermore, a seek + sync that sets the position outside the loop can be corrected to be inside.

Reply:
Yes, even better readability. Thank you.

But why do we need syncPosition outside the if block? 
I think it would be even more clean to have a requested position as well and not reuse position for this.